### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by npanpaliya**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,42 @@ For example:
...
 
+### Using Open-CE python with other Open-CE packages
+
+One can use Open-CE packages along with the optimized Open-CE Python. Following are the steps to setup Open-CE Python and Open-CE packages in a single conda environment -
+
+1. Create a new conda environment with optimized python built as mentioned in above section.
+```shell
+    conda create -n <conda env name> python=3.10.9 -c file://<path to condabuild directory>
+```
+   Here, `path to condabuild directory` means the path to `--output_folder` when python was built as per above section.
```
Comment (alexlang74) [inline README.md:51]: the `open-ce build env` command doesn't include the `--output_folder` attribute. Do you want to add it?

File: README.md [hunk 2]
```diff
@@ -40,6 +40,42 @@ For example:
...
+   By default it is `condabuild` named directory.
```
Comment (alexlang74) [inline README.md]: If you have not specified `--output-folder` , the directory will be `condabuild` .

File: README.md [hunk 3]
```diff
@@ -40,6 +40,42 @@ For example:
...
+#
+# Name                    Version                   Build  Channel
+python                    3.10.9               hd70dfa7_0    file://<path to condabuild directory>
+```
+
+4. Install Open-CE packages from any of the following channels -
+* https://ftp.osuosl.org/pub/open-ce/current/
+* https://opence.mit.edu/
+* https://anaconda.org/rocketce/
+
```
Comment (alexlang74) [inline README.md:73]: please move rocketce to the top

File: README.md [hunk 4]
```diff
@@ -40,6 +40,42 @@ For example:
...
+For example:
+```shell
+conda install -y tensorflow-cpu libtensorflow onnx onnxruntime pytorch-cpu openblas -c rocketce
+```
+One must ensure that python installed in the conda environment is not updated/replaced by any other python source like Anaconda. Step #3 can be repeated here to ensure this.
```
Comment (alexlang74) [inline README.md]: 5. Python verification

When you're done with the package installation, run `(<conda env name>)$ conda list python` again to ensure that the python installed in the conda environment is not updated/replaced by any other python source like Anaconda

File: README.md (hunk 1 above)
Comment (npanpaliya) [reply README.md:51]: Yes, okay.

File: README.md (hunk 2 above)
Comment (npanpaliya) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (npanpaliya) [reply README.md:73]: Okay

File: README.md (hunk 4 above)
Comment (npanpaliya) [reply README.md]: Sure

File: README.md [hunk 5]
```diff
@@ -40,6 +40,42 @@ For example:
...
+(<conda env name>)$ conda list python
+# packages in environment at $HOME/anaconda3/envs/<conda env name>:
+#
+# Name                    Version                   Build  Channel
+python                    3.10.9               hd70dfa7_0    file://<path to condabuild directory>
+```
+
+4. Install Open-CE packages from any of the following channels -
+* https://ftp.osuosl.org/pub/open-ce/current/
+* https://opence.mit.edu/
```
Comment (cdeepali) [inline README.md]: Can we indent these bullets?

Comment (npanpaliya) [reply README.md]: Done

**Review by alexlang74 — APPROVED** [review]

**Pull request merged**